compute_tools/src/sql/finalize_drop_subscriptions.sql

```diff
@@ -0,0 +1,21 @@
+DO $$
+BEGIN
+    IF NOT EXISTS(
+        SELECT 1
+        FROM pg_catalog.pg_tables
+        WHERE tablename = 'drop_subscriptions_done'
+        AND schemaname = 'neon_migration'
+    )
+    THEN
+        CREATE SCHEMA IF NOT EXISTS neon_migration;
+        ALTER SCHEMA neon_migration OWNER TO cloud_admin;
+        REVOKE ALL ON SCHEMA neon_migration FROM PUBLIC;
+
+        CREATE TABLE neon_migration.drop_subscriptions_done
+        (timeline_id text);
```

Comment:
Not sure where exactly, but I'd mention in the comments why we store the timeline_id here, i.e. that during time-travel and branch resets we may need to do the drop again, so we cannot rely on just boolean flag as it might be already set

Reply:
added the comment here, but there is also big comment in `apply_spec_sql`

Reply:
> but there is also big comment in apply_spec_sql

Yeah, that was good, thanks, it just didn't mention why we need to store timeline id